File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.cs
using System;

public class Solution 
{
    public string solution(string my_string, int[] indices) 
    {
        string answer = "";
        Array.Sort(indices);
        bool isBool = false;
        
        for (int i=0; i<my_string.Length; i++) {
            foreach(int indice in indices) {
                if(i == indice) {
                    isBool = true;
                }
            }
            
            if (!isBool) {
                answer += my_string[i];   
            }
            
            isBool = false;
        }
        return answer;
    }
}

File: 프로그래머스/0/181900. 글자 지우기/글자 지우기.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>

using namespace std;

string solution(string my_string, vector<int> indices) 
{
    sort(indices.begin(), indices.end()); // 오름차순 정렬
    for(int i=0; i<indices.size(); i++)
    {
        my_string.erase(my_string.begin() + indices[i]);
        for(int j = i + 1; j<indices.size(); j++)
        {
            indices[j]--;
        }
    }
    return my_string;
}